config/search_form.yml

```diff
@@ -0,0 +1,45 @@
+default: &default
```

Comment:
Feel free to make another ticket for this, but it could be nice to have something in the README and/or comments in the code explaining how to configure the search form.